### DISCUSSION TITLE
Transmission web ui not displayed; curl returns unauthorised

### DISCUSSION THREAD

**Discussion by ncharlez** [haugene/docker-transmission-openvpn#2822] [Q&A]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [x] I have tried using dev branch

### Docker run config used

transmission-vpn:
    image: haugene/transmission-openvpn:latest
    container_name: transmission-vpn
    restart: always
    environment:
     - OPENVPN_PROVIDER=CUSTOM
     - OPENVPN_CONFIG=AirVPN_Czech-Republic_UDP-443-Entry3
     - OPENVPN_USERNAME=notused
     - OPENVPN_PASSWORD=notused
     - OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60
     - LOCAL_NETWORK=192.168.0.0/24
     - PUID=1000
     - PGID=1000
     - TZ=${TZ}
     - TRANSMISSION_RPC_AUTHENTICATION_REQUIRED=true
     - TRANSMISSION_RPC_HOST_WHITELIST="127.0.0.1,192.168.*.*"
     - TRANSMISSION_RPC_PASSWORD=XXXX
     - TRANSMISSION_RPC_USERNAME=XXXX
     - TRANSMISSION_RATIO_LIMIT=0
     - TRANSMISSION_RATIO_LIMIT_ENABLED=true
    volumes:
     - ${USERDIR}/docker/VPNCzech:/etc/openvpn/custom
     - ${USERDIR}/docker/transmission-vpn:/data
     - ${USERDIR}/docker/shared:/shared
     - ${USERDIR}/Downloads:/data/watch
     - ${USERDIR}/Downloads/complete:/data/completed
     - ${USERDIR}/Downloads/incomplete:/data/incomplete
    ports:
     - "9107:9091"
     # - "9106:9117" 
     # - "9111:9696"
    dns: 
     - 8.8.8.8
     - 1.0.0.1    
    cap_add:
     - NET_ADMIN
    devices:
     - /dev/net/tun
    sysctls:
     - net.ipv6.conf.all.disable_ipv6=1
     - net.ipv6.conf.default.disable_ipv6=1
     - net.ipv6.conf.lo.disable_ipv6=1

### Current Behavior

Running Docker Desktop on a MacBook Pro.
Transmission used to work on Docker Desktop, but I used colima for a while and just switched back to to Docker Desktop.
Running the same compose.yaml configuration that has always worked previously, transmission seems to start correctly, but never displays the web ui. 

Following the instruction in the 'Debugging your setup' section from https://haugene.github.io/docker-transmission-openvpn/debug/, when I exec into the container and run curl localhost:9091, it does not return '<h1>301: Moved Permanently</h1>' as expected. Instead, it returns '<h1>401: Unauthorized</h1>' . I get the same message if I run: curl localhost:9091/transmission/web.

### Expected Behavior

I expect when I run 'curl localhost:9091' to get the <h1>301: Moved Permanently</h1> as specified in the documentation. 

### How have you tried to solve the problem?

Ran checks described in the documentation.

### Log output

[untitled text.txt](https://github.com/haugene/docker-transmission-openvpn/files/14762228/untitled.text.txt)

### HW/SW Environment

```markdown
- OS: Mac osX Monterey (12.7.3)
- Docker: 25.0.3 build 4debf41
- Docker Desktop 4.28
```

### Anything else?

I didn't used to have the ipv6 exclusions in the compose file. They were added recently when I kept getting ipv6 errors.

**Comment by ncharlez** [comment]
Downgraded to 5.0.0 and the same thing happens

**Comment by pkishino** [comment]
There are some issues in the log, you haven’t disabled IPv6 correctly per the wiki and some permissions are failing..
But the issue you are having is transmission related so please check their report for support and check that wiki. 
I’d suggest using the basic compose from our wiki to check basics before adding rpc etc
